python/tempo/utils.py

```diff
@@ -57,19 +57,29 @@ def display_unavailable(df):
 ENV_BOOLEAN = __isnotebookenv()
 
 if PLATFORM == "DATABRICKS":
-    method = get_ipython().user_ns['display']
 
-
-    # Under 'display' key in user_ns the original databricks display method is present
-    # to know more refer: /databricks/python_shell/scripts/db_ipykernel_launcher.py
-    def display_improvised(obj):
-        if type(obj).__name__ == 'TSDF':
-            method(obj.df)
-        else:
-            method(obj)
-
-
-    display = display_improvised
+    # This below check is for ensuring compatibility with Databricks DLT runtimes
+    # This if logic ensures that the custom user's namespace of DLT runtimes
+    # which doesn't have PythonShell's Display object in the namespace doesn't result in an error.
+    if (
+            ('create_dlt_table_fn' not in list(get_ipython().user_ns.keys()))
+            and
+            ('dlt_sql_fn' not in list(get_ipython().user_ns.keys()))
```

Comment:
You're performing this check multiple times, you should use a function.
Maybe create a function in `__init__.py` called something like `__is_display_fn_avail()` where you have this check.

Simple rule to follow with your code: any time you find yourself writing/copying the same code - it's time for a function (or global var)

Reply:
Yes makes sense I will move this to a function

Reply:
made this into a function